### PR TITLE
ctl: print durations in get_region_read_progress

### PR DESCRIPTION
**Pull request by ekexium**
### What is changed and how it works?

Issue Number: ref #15082 

What's Changed:

```commit-message
Record and print `duration to last update_safe_ts` and `duration to last consume leader`.
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Side effects

- Performance regression
    - Consumes more CPU

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/store/util.rs [hunk 1]
```diff
@@ -1200,10 +1200,11 @@ impl RegionReadProgressRegistry {
     ) -> Vec<u64> {
         let mut regions = Vec::with_capacity(leaders.len());
         let registry = self.registry.lock().unwrap();
+        let now = Some(Instant::now());
```
Comment (cfzjywxk) [inline components/raftstore/src/store/util.rs:1203]: As mentioned in the above comments, would it introduce much performance overhead considering the frequency of checking leaders, or should we use `now_coarse()` instead?

File: cmd/tikv-ctl/src/executor.rs [hunk 2]
```diff
@@ -1008,15 +1008,20 @@ impl DebugExecutor for DebugClient {
...
-            // (
-            //     "duration to last update_safe_ts",
-            //     format!("{} ms", resp.get_duration_to_last_update_safe_ts_ms()),
-            // ),
-            // (
-            //     "duration to last consume_leader_info",
-            //     format!("{} ms", resp.get_duration_to_last_consume_leader_ms()),
-            // ),
+            (
+                "duration to last update_safe_ts",
```
Comment (MyonKeminta) [inline cmd/tikv-ctl/src/executor.rs:1012]: How many places might update them without the `now` parameter? Should we consider call it "approximate duration to ..."?

**Review by MyonKeminta — COMMENTED** [review]
LGTM

File: components/raftstore/src/store/util.rs (hunk 1 above)
Comment (ekexium) [reply components/raftstore/src/store/util.rs:1203]: I believe this is acceptable by estimating the impact: If there are 1M active regions, interval of check leader is 5s, a `now()` cost ~20ns in my testing Linux environment. That would be 0.02s/5s = <1% CPU usage.
`now_coarse` sounds good.

File: cmd/tikv-ctl/src/executor.rs (hunk 2 above)
Comment (ekexium) [reply cmd/tikv-ctl/src/executor.rs:1012]: As far as I can see, they're only cdc and backup-stream. Let's call it "... from resolved-ts routine"

**Review by MyonKeminta — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**